src/kiwixapp.h

```diff
@@ -16,6 +16,8 @@
 #include <QApplication>
 #include <QErrorMessage>
 #include <QTranslator>
+#include <kiwix/name_mapper.h>
+#include <kiwix/tools.h>
```

Comment:
Why do you need to include tools ?

Reply:
Seems useless indeed. Compile and run properly after my rebase and removal.